content/resources/tutorials/contents.lr

```diff
@@ -1,11 +1,92 @@
-title: Tutorials
----
-body:
+
+dy:
 
 # Tutorials
 
 - [Setting up SSH with Garfield (LABnet)](/resources/tutorials/garfield_ssh)
 
-## Ideas for future articles
+- Web Development:
+  * [w3schools](https://www.w3schools.com/) 
+  * [HTML in 30 minutes or less!](https://www.youtube.com/watch?v=hrZqiCUx6kg)
+  * [JavaScript playlist](https://youtu.be/fGdd9qNwQdQ?list=PLoYCgNOIyGACnrXwo5HMCfOH9VT05znGv)
+  * [ReactJS basics](https://youtu.be/JPT3bFIwJYA?list=PL55RiY5tL51oyA8euSROLjMFZbXaV7skS)
+  * [Angular 5 in 60 Minutes](https://youtu.be/oa9cnWTpqP8)
+
+- Mobile Development:
+  * Android -
+    * [Kotlin](https://www.youtube.com/watch?v=W99WA5YoAp0&index=1&list=PLbfTjwmfwhuhPRMOUT3RMC-naLWK6RJ62)
+    * [Java](https://www.youtube.com/watch?v=sZWMPYIkNd8)
+  * iOS -
+    * [Swift](https://www.youtube.com/watch?v=5b91dFhZz0g)
+    * [Objective C](https://www.youtube.com/watch?v=5esQqZIJ83g)
+  * [React Native (React based)](https://www.youtube.com/watch?v=mkualZPRZCs)
+  * [NativeScript (Angular based)](https://www.youtube.com/watch?v=n0RZbTGf6uQ)
+
+- Full Stack:
+  * [Definition](https://youtu.be/jPSo7Lge6bA)
+  * [MEAN stack](https://www.youtube.com/watch?v=PFP0oXNNveg)
+  * [MERN stack](https://www.youtube.com/watch?v=QA29Qvr9rdw&list=PLm7Ikk48Q5FzGVpLV4GR9e5bG0c5bUvQ1)
+
+- Virtual Reality:
+  * [Various videos](https://www.youtube.com/watch?v=Pgvpj63823Q&list=PLmE2ibStnoYoarCS2G7OMtGOdslvl8hx0)
+
+- [Arduino](https://www.youtube.com/watch?v=d8_xXNcGYgo&list=PLGs0VKk2DiYx6CMdOQR_hmJ2NbB4mZQn-)
+
+- [Raspberry Pi](https://www.youtube.com/watch?v=Jj4pjfU_-jo&list=PLsa31gkyINsly6N_usaeHrtDPYnwxO-1Y)
+
+- Machine Learning 
+  * [The famous Stanford University course by Andrew Ng!](https://www.youtube.com/watch?v=PPLop4L2eGk&list=PLLssT5z_DsK-h9vYZkQkYNWcItqhlRJLN)
+  * [Machine Learning Recipes by Google Developers](https://www.youtube.com/watch?v=cKxRvEZd3Mw&list=PLOU2XLYxmsIIuiBfYad6rFYQU_jL2ryal)
+  * [Neural Networks with Siraj](https://www.youtube.com/watch?v=h3l4qz76JhQ&list=PL2-dafEMk2A5BoX3KyKu6ti5_Pytp91sk)
```

Comment:
This guy <3

Reply:
lol, I was thinking you could add the slides for your NDev presentation here as well. I should have added my _Web Scraping with Python_ talk.